readthedocs/settings/docker_compose.py

```diff
@@ -155,7 +155,7 @@ def show_debug_toolbar(request):
     # Storage backend for build languages
     RTD_BUILD_TOOLS_STORAGE = 'readthedocs.storage.s3_storage.S3BuildToolsStorage'
     # Storage for static files (those collected with `collectstatic`)
-    STATICFILES_STORAGE = 'readthedocs.storage.s3_storage.S3StaticStorage'
+    STATICFILES_STORAGE = 'storages.backends.s3boto3.S3ManifestStaticStorage'
```

Comment:
We shouldn't use this backend directly. Instead, we should keep using the one we were using but inheriting from this one that support manifest instead.

In our own backed we are using other stuff as well.